EDI/UserInterface/03a_LISTLogging.md

```diff
@@ -0,0 +1,163 @@
+# LIST (Log)
+
+*DataHub Exporter > Log > Tab LIST*
+
+![Log](../../Assets/Screenshots/EDI/UserInterface/Log.png "[Log]")
+
+**List of intents**   
+The *List of intents* contains all intents. An intent is a summary of log entries generated by one system action. Depending on the settings, the displayed columns may vary. All fields are read-only.
+
```

Comment:
Keine Buttons in editing toolbar? [SHOW MESSAGE]?

Reply:
Dane